server/migrations/20210810194844_populate_area_changes.js

```diff
@@ -0,0 +1,10 @@
+const populatePlanAreaChanges = require('../bin/populate_plan_area_changes');
+
+
+exports.up = async function(knex) {
+  await populatePlanAreaChanges();
+};
+
+exports.down = function(knex) {
+  knex.raw('DELETE FROM plan_area_changes');
```

Comment:
Can you please use 
```
 await knex.schema.dropTableIfExists('plan_area_changes');
```

Reply:
It shouldn't drop the table, it should remove all the entries. I changed this line to be more Javascript-y and that will delete all the rows.

Reply:
Where's this table is created from? When it is created?

Reply:
One migration before this one. It is created in a previously accepted PR.

Reply:
oh ok then